components/resc-backend/setup.cfg

```
@@ -11,7 +11,28 @@ long_description_content_type = text/markdown
 
 [options]
 python_requires = >=3.9
-include_package_data = False
+install_requires =
+    celery==5.3.1
+amqp==5.1.1
+requests==2.31.0
+sqlalchemy==2.0.20
+alembic==1.11.2
+fastapi==0.103.0
+typing==3.7.4.3
+pydantic==1.8.2
+uvicorn==0.17.6
+waitress==2.1.2
+pyjwt[crypto]==2.8.0
+cryptography==41.0.3
+tenacity==8.2.2
+tomlkit==0.12.1
+python-multipart==0.0.6
+aiofiles==0.8.0
+GitPython==3.1.32
+packaging==23.1
+fastapi-cache2==0.2.1
+redis==4.6.0
+include_package_data = True
```

```suggestion
include_package_data = True
```
Is this change necessary? (similar for the other occurrences)